packages/elements/src/components/ui/collapsible/Collapsible.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from "react";
+import { ReactNode } from "react";
+import { config, Spring } from "react-spring/renderprops-universal";
+import styles from "./Collapsible.module.css";
+import { Icon } from "../../..";
+import { faChevronDown } from "@fortawesome/free-solid-svg-icons";
+
+export interface CollapsibleProps {
+  label: string;
+  contentRight?: ReactNode;
+  collapsed?: boolean;
+  onCollapse?: () => void;
```

Comment:
This should probably be `onClick` instead, to be consistent with all other components.